src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.component.js

```diff
@@ -130,6 +156,13 @@ class ExistingPaymentMethodsController {
 
   switchPayment () {
     this.onPaymentChange({ selectedPaymentMethod: this.selectedPaymentMethod })
+    if (this.selectedPaymentMethod?.['card-type'] && this.creditCardPaymentForm?.securityCode) {
+      // Set cvv from session storage
+      const storage = JSON.parse(this.sessionStorage.getItem('cvv')) || ''
+      this.creditCardPaymentForm.securityCode.$setViewValue(storage)
+      this.creditCardPaymentForm.securityCode.$render()
+      this.sessionStorage.removeItem('cvv')
```

Comment:
We don't want to clear this from session storage here, as this is what is sent to the server.

Reply:
In my testing, it cleared right after I edited the existing payment method and entering the CVV there. It got copied to the text box but then removed from session storage.

Reply:
Ah, I see what you're saying. It does get added back after hitting continue. We need to test that on the single step BCO branch as well to see how that behaves.

Reply:
Actually, the one step BCO won't have this issue due to the user being logged out and not being able to go back from a review step to edit, the 2 step BCO will, so I can try testing there.

Reply:
So, it looks like `switchPayment` is called when hitting `Continue` on 2 step branded checkout and when hitting `Submit` on single step branded checkout. So this will definitely need to change in some way.